src/actions/arcActions.ts

```diff
@@ -587,7 +587,7 @@ export function createProposal(daoAvatarAddress: string, title: string, descript
       const schemeParamsHash = await dao.controller.getSchemeParameters(contributionRewardInstance.contract.address, dao.avatar.address);
       const schemeParams = await contributionRewardInstance.contract.parameters(schemeParamsHash);
       const votingMachineAddress = schemeParams[2];
-      const votingMachineInstance = await Arc.GenesisProtocolFactory.at(votingMachineAddress);
```

Comment:
we can remove the votingMachineAddress const no?

Reply:
done https://github.com/daostack/alchemy/pull/103/commits/e6e5ebf9f63cfa24e4c88bb950ee25afcc9f9e51